less/index.less

```diff
@@ -3,57 +3,344 @@
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+}
+/* my code */
+//import colors 
+@import "colors.less";
+@import "font.less";
+@import "mixins.less";
+
+/*style for container */
+.container {
+  max-width: 880px;
+  margin: 0 auto;
+ border: 1px solid @transparentWhite;
+  background: @transparentWhite;
+  box-shadow: 0px 0px 2px 2px @shadowColor;
+}
+.intro {
+  position: relative;
+  box-shadow: 0px 0px 2px 3px @shadowColor;
+}
+.overlay {
+  position: absolute;
+  background: @overlay;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  bottom: 0;
+}
+.header {
+  background: url(/image/img3.jpg) no-repeat;
+  width: 100%;
+  height: 50vh;
+  background-size: cover;
+  background-position: center;
+
+  .nav-links {
+   .flex-display(row, flex-end);
+    width: 100%;  
+
+    a {
+      .flex-display(row, flex-start);
+      z-index: 1;
+      color: @white;
+      width: 10%;
+      text-decoration: none;
+      text-transform: capitalize;
+      font-size: @mediumFontSize;
+      padding-top: 1rem;
+    }
+  }
+  .intro-text {
+    .flex-display(column, center, center);
+    height: 80%;
+
+    h1 {
+      font-size: @h1fontSize;
+      color: @white;
+      z-index: 0;
+    }
+    p {
+      font-size: @h3fontSize;
+      color: @transparentWhite;
+      z-index: 1;
+      padding-top: 2rem;
+    }
+  }
+}
+.personal-info {
+  background: @white;
+  width: 100%;
+  display: flex;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  padding-left: 2rem;
+  padding-right: 1.5rem;
+  box-shadow: 0px 0px 2px 2px @shadowColor;
+
+  .about-me {
+    width: 50%;
+
+    h3 {
+      font-size: 3rem;
+      padding-bottom: 2rem;
+    }
+    p {
+      font-size: @mediumFontSize;
+      line-height: 1.2;
+    }
+  }
+
+  .about-me-img {
+    width: 40%;
     box-sizing: border-box;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+
+    img {
+      width: 200px;
+      height: 200px;
+	  border-radius: 80%;
+	  border: 1px solid @inputBorder;
+    }
+  }
 }
+.work-info {
+  display: flex;
+  padding: 2rem;
+  margin-top: 3rem;
+  justify-content: center;
+  background: @white;
+  box-shadow: 0px 0px 2px 2px @shadowColor;
+
+  .skills {
+    width: 50%;
+
+    h3 {
+      font-size: @h3fontSize;
+      padding-bottom: 2rem;
+    }
+
+    ul {
+      font-size: @mediumFontSize;
+      list-style-type: square;
+      padding-left: 1.5rem;
+      line-height: 1.5;
 
-html, body {
-    height: 100%;
+      li {
+        text-transform: uppercase;
+        border-radius: 2px;
+      }
+    }
+  }
+
+  .work-history {
+    width: 50%;
+    padding-left: 6.5rem;
+
+    h3 {
+      font-size: @h3fontSize;
+      padding-bottom: 2rem;
+    }
+    h4 {
+      font-weight: bold;
+      font-size: @mediumFontSize;
+    }
+    p {
+      font-size: @mediumFontSize;
+      padding-bottom: 2rem;
+    }
+  }
 }
 
+.contact {
```

Comment:
All the section below is fine, but I feel it's nested too deeply.
It's hard too read, you could want to refactor it with cleaner code.